npm-shrinkwrap.json

```diff
@@ -3845,6 +3840,11 @@
         }
       }
     },
+    "openid": {
+      "version": "0.5.13",
+      "from": "openid@0.5.13",
+      "resolved": "https://registry.npmjs.org/openid/-/openid-0.5.13.tgz"
+    },
```

Comment:
When we come to deploy this for real, we should consider investing in a security audit of this library, unless they've already had a public one done.